Index: nhd2-exp/configure.ac
===================================================================
--- nhd2-exp.orig/configure.ac
+++ nhd2-exp/configure.ac
@@ -18,16 +18,16 @@ TUXBOX_APPS_LIB_CONFIG(CURL,curl-config)
 TUXBOX_APPS_LIB_CONFIG(FREETYPE,freetype-config)
 TUXBOX_APPS_LIB_PKGCONFIG_CHECK(ID3TAG,libid3tag)
 if test x"$ID3TAG_EXISTS" != xyes; then
-	TUXBOX_APPS_LIB_PKGCONFIG(ID3TAG,id3tag)
+	TUXBOX_APPS_LIB_PKGCONFIG_CHECK(ID3TAG,id3tag)
 fi
 TUXBOX_APPS_LIB_PKGCONFIG_CHECK(MAD,libmad)
 if test x"$MAD_EXISTS" != xyes; then
-	TUXBOX_APPS_LIB_PKGCONFIG(MAD,mad)
+	TUXBOX_APPS_LIB_PKGCONFIG_CHECK(MAD,mad)
 fi
 TUXBOX_APPS_LIB_PKGCONFIG(PNG,libpng)
-TUXBOX_APPS_LIB_PKGCONFIG(AVCODEC,libavcodec)
-TUXBOX_APPS_LIB_PKGCONFIG(AVUTIL,libavutil)
-TUXBOX_APPS_LIB_PKGCONFIG(AVFORMAT,libavformat)
+TUXBOX_APPS_LIB_PKGCONFIG_CHECK(AVCODEC,libavcodec)
+TUXBOX_APPS_LIB_PKGCONFIG_CHECK(AVUTIL,libavutil)
+TUXBOX_APPS_LIB_PKGCONFIG_CHECK(AVFORMAT,libavformat)
 
 # keybaord
 AC_ARG_ENABLE(keyboard-no-rc,
