From 984d0d1977d70602b3d877c1526c60613a8c8449 Mon Sep 17 00:00:00 2001
From: Christian Ege <k4230r6@googlemail.com>
Date: Wed, 1 May 2013 21:24:44 +0200
Subject: [PATCH] fixed framebuffer device

---
 tuxcom/tuxcom.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tuxcom/tuxcom.c b/tuxcom/tuxcom.c
index 0134ed9..2ed690d 100644
--- a/tuxcom/tuxcom.c
+++ b/tuxcom/tuxcom.c
@@ -623,7 +623,7 @@ int main()
 		ey = atoi(env);
 
 	/* open Framebuffer */
-	fb=open("/dev/fb/0", O_RDWR);
+	fb=open("/dev/fb0", O_RDWR);
 	if (fb < 0) {
 		perror("TuxCom <open framebuffer>");
 		exit(1);
-- 
1.7.7

