From 147d94a2b99c378f1c3d614cc193f8131f0d8343 Mon Sep 17 00:00:00 2001
From: graugans <k4230r6@gmail.com>
Date: Wed, 6 Feb 2013 06:24:29 +0100
Subject: [PATCH] fixed nhttpd web root path

---
 src/nhttpd/nhttpd.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/nhttpd/nhttpd.conf b/src/nhttpd/nhttpd.conf
index de95579..e35e503 100644
--- a/src/nhttpd/nhttpd.conf
+++ b/src/nhttpd/nhttpd.conf
@@ -1,7 +1,7 @@
 Language.directory=languages
 Language.selected=Deutsch
 Tuxbox.LogosURL=/usr/share/tuxbox/neutrino/icons/logo
-WebsiteMain.directory=/share/tuxbox/neutrino/httpd
+WebsiteMain.directory=/var/tuxbox/neutrino/httpd
 WebsiteMain.override_directory=/var/httpd
 WebsiteMain.port=80
 WebsiteMain.hosted_directory=/var/hosted
-- 
1.7.7

