Index: vsftpd-2.3.5/Makefile
===================================================================
--- vsftpd-2.3.5.orig/Makefile
+++ vsftpd-2.3.5/Makefile
@@ -6,7 +6,6 @@ IFLAGS  = -idirafter dummyinc
 CFLAGS	=	-O2 -Wall -W -Wshadow #-pedantic -Werror -Wconversion
 
 LIBS	=	-lssl -lcrypto -lnsl -lresolv
-LINK	=	-Wl,-s
 
 OBJS	=	main.o utility.o prelogin.o ftpcmdio.o postlogin.o privsock.o \
 		tunables.o ftpdataio.o secbuf.o ls.o \
